src/index.tsx

```diff
@@ -37,23 +62,48 @@ const FAVORITE_ITEM_PINNER_CLASS = 'jp-Favorites-pinner';
  */
 const BREADCUMBS_FAVORITES_CLASS = 'jp-Favorites-crumbs';
 
+/**
+ * Cell tag used to mark cell as favorite
+ */
+const FAVORITE_TAG = 'favorite';
+
+/**
+ * Class set to notebook when filtering cells by favorite is enabled
+ */
+const FAVORITE_FILTER_CLASS = 'jp-favorites-filter-active';
+
+/**
+ * Setting key for toggling the visibility of star icons on all cells.
+ */
+const SHOW_STARS_ON_ALL_CELLS = 'showStarsOnCells';
```

Comment:
This is a more general setting now:


```suggestion
const SHOW_STARS_ON_CELLS = 'showStarsOnCells';
```